src/Components/Admins/Subscriptions/Table/index.jsx

```diff
@@ -70,7 +105,7 @@ const Table = ({ data }) => {
                 )}
                 <td>{formatDate(subscription.date)}</td>
                 <td className={`${styles.itemButton} ${styles.itemButtonEdit}`}>
-                  <Link to={`subscriptions/edit/${subscription._id}`}>
+                  <Link to={`/subscriptions/edit/${subscription._id}`}>
```

Comment:
this must be
`<Link to={`/user/admin/subscriptions/edit/${subscription._id}`}>`
the previous path no longer works